WooCommerce/Classes/ViewRelated/Orders/Order Creation/NewOrderViewModel.swift

```diff
@@ -63,6 +65,14 @@ final class NewOrderViewModel: ObservableObject {
         return statusResultsController.fetchedObjects
     }
 
+    // MARK: Products properties
+
+    /// View model for the product list
+    ///
+    lazy var addProductViewModel = {
+        AddProductToOrderViewModel(siteID: siteID, storageManager: storageManager)
+    }()
```

Comment:
This looks to be working now, but you need to be aware that any state generated inside `AddProductToOrderViewModel` will be maintained and could give you problems as the VM grows with functionality!